src/mjolnir/pbfgraphparser.cc

```diff
@@ -1030,6 +1037,74 @@ struct graph_callback : public OSMPBF::Callback {
       }
     }
 
+    if (use_direction_on_ways_ && !ref.empty()) {
```

Comment:
do we have any tests for the direction assignments?

Reply:
will add some